src/Language/PureScript/Parser/Declarations.hs

```diff
@@ -334,9 +334,14 @@ parseObjectLiteral :: TokenParser Expr
 parseObjectLiteral = ObjectConstructor <$> braces (commaSep parseIdentifierAndValue)
 
 parseIdentifierAndValue :: TokenParser (String, Maybe Expr)
-parseIdentifierAndValue = (,) <$> (C.indented *> (lname <|> stringLiteral) <* C.indented <* colon)
-                              <*> (C.indented *> val)
+parseIdentifierAndValue =
+  do
+    name <- lname
+    b <- P.option (Just $ Var $ (Qualified Nothing) (Ident name)) rest
+    return (name, b)
+  <|> (,) <$> (C.indented *> (lname <|> stringLiteral)) <*> rest
```

Comment:
Ah! The first branch handles the cases of `lname` with and without a value!

Updating the PR. Thanks!